css/style.css

```diff
@@ -17,3 +17,325 @@ body {
+  font-size:52px;
+  color:black;
+  padding-top:10rem;
+  padding-left:27rem;
+ }
+
+
+   /* FLEXBOX for three icons*/
+
+   .iconFlexBox {
+      margin-top:4rem;
+      margin-left:16rem;
+      display: flex;
+      gap:7rem;    
+      padding-bottom:6rem;
+
+  }
+
+   .description {
+    font-size:28px;
+   }
+
+   .iconPadding{
+    padding-left:4rem;
+    padding-bottom: 3rem;
+   }
+
+   .iconPadding2{
+    padding-left:2rem;
+    padding-bottom: 3rem;
+   }
+   
+
+
+/* Footer */
+
+ .theFooter {
+  margin-left: 7.3rem;
+  margin-right: 7.3rem;
+  position: absolute;
+  right: 0;
+  bottom: 14;
+  left: 0;
+  padding: 1rem;
+  border-top-color:gray;
+  background-color: white;
+  text-align: center;
+ }
+  
+ 
+    /* Based on https://www.w3.org/Style/Examples/007/center.en.html */
+
+  .placeInTheCentre {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%)
+  }
+
+     /* FLEXBOX for three social media icons*/
+     .socialMediaFlexBox {
+      display: flex;
+      gap:1rem;    
+    } 
+
+  .socialMedia { 
+    object-fit: cover;  
+    
+    /* Circular Border 
+    see https://css-tricks.com/almanac/properties/b/border-radius/
+    For a circle, 50% */
+
+    border-radius: 50%;
+    /* --grey-light: #e4e4e4; */
+    border: 1px solid #e4e4e4;;
+    padding:1rem;
+    } 
+
+    .drawaline
+      {width: 70%;
+        padding-right: 5rem;
+        margin-bottom: 2rem; 
+       /* --grey-light: #e4e4e4; */
+       background-color:#e4e4e4;
+       height:1px; /* Must set the height to show the coloured line -see https://www.tutorialrepublic.com/faq/how-to-change-the-color-of-an-hr-element-using-css.php */
+          }
+      
+    .joinUs {
+        font-size: 22px;
+         /* padding-bottom: 8em; */
+      margin-bottom: 8em;
+    }
+  
+    .copyright {
+      position: absolute;
+      top: 70%;
+      width: 100%;
+      left: 10px;
+      font-size: 20px;
+      color: gray;
+    }
+/*
+I know this may be a bit of an overkill, however I noticed that on 
+https://raw.githubusercontent.com/CodeYourFuture/HTML-CSS-Coursework-Week2/master/Karma%20-%20desktop%20design.png
+
+the instagram icon is actually BLUE!
+Is there a way then to change the colour from orange to blue? Yes!
+
+This solution is taken from 
+    https://codepen.io/sosuke/pen/Pjoqqp 
+which in turn is based on 
+    https://stackoverflow.com/a/43960991/604861 
+*/
+
+
+    .blueInstagram { 
+      object-fit: cover;  
+
+      /* For this code to work well the starting color needs to be black. 
+      If your icon set isn't black you can prepend "brightness(0) saturate(100%)" to your filter property 
+      which will first turn the icon set to black. */
+
+      filter: brightness(0) saturate(100%);
+
+      /* Now convert to Instagram Blue, that is, the target colour is #405DE6; */
+      filter: invert(40%) sepia(78%) saturate(481%) hue-rotate(181deg) brightness(83%) contrast(85%);
```

Comment:
Clever! But as you say a bit overkill. Better to tweak the SVG itself to change the colour, though that is more complex.